bucket/plex-desktop.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.29.1.1974-a65b3ef3",
+    "description": "Plex for Windows.",
+    "homepage": "https://www.plex.tv/apps-devices/#modal-devices-plex-for-windows",
+    "license": "Proprietary",
```

Comment:
https://www.plex.tv/media-server-downloads/?platform=windows#remodal-terms

Reply:
you mean to change Proprietary to the URL?